lib/utils/otel/metric.ts

```diff
@@ -0,0 +1,66 @@
+import { Resource } from '@opentelemetry/resources';
+import { PrometheusExporter, PrometheusSerializer } from '@opentelemetry/exporter-prometheus';
+import { SEMRESATTRS_SERVICE_NAME } from '@opentelemetry/semantic-conventions';
+import { MeterProvider } from '@opentelemetry/sdk-metrics';
+import { Attributes } from '@opentelemetry/api';
+
+interface IMetricAttributes extends Attributes {
+    method: string;
+    path: string;
+    status: number;
+}
+
+interface IHistogramAttributes extends IMetricAttributes {
+    unit: string;
+}
+
+const metric_prefix = 'rsshub';
+
+const exporter = new PrometheusExporter({});
+
+const provider = new MeterProvider({
+    resource: new Resource({
+        [SEMRESATTRS_SERVICE_NAME]: 'rsshub',
+    }),
+    readers: [exporter],
+});
+
+const serializer = new PrometheusSerializer();
+
+const meter = provider.getMeter('rsshub');
+
+const request_total = meter.createCounter<IMetricAttributes>(`${metric_prefix}_request_total`);
+const request_error_total = meter.createCounter<IMetricAttributes>(`${metric_prefix}_request_error_total`);
+const request_duration_seconds_bucket = meter.createHistogram<IHistogramAttributes>(`${metric_prefix}_request_duration_seconds_bucket`, {
+    advice: {
+        explicitBucketBoundaries: [0.01, 0.1, 1, 2, 5, 15, 30, 60],
+    },
+});
+const request_duration_milliseconds_bucket = meter.createHistogram<IHistogramAttributes>(`${metric_prefix}_request_duration_milliseconds_bucket`, {
+    advice: {
+        explicitBucketBoundaries: [10, 20, 50, 100, 250, 500, 1000, 5000, 15000],
+    },
+});
+
+export const metric_handler = {
```

Comment:
```suggestion
export const metric = {
```

Reply:
I'm not sure if the rename is necessary, the `metric_handler` here is just a wrapper around the aggregated metrics counter

Reply:
Or any other better var name?

Reply:
[Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard#naming)